### ISSUE TITLE
Final issues before presentation (to fix later)

### ISSUE THREAD

**Issue by camille-004** [camille-004/group-testing-rl#2]
- Log "`pretraining...`" when `win_rate = 0.0`, before `n_epochs == history_window_size`
- Find a better way to fix repetition issue, rather than agent learning not to repeat, as this adds unnecessary training time and does not satisfy the constraints of the original problem
- More seamless ways to tune training hyperparameters

- Change maximum epsilon threshold when win rate seems to stagger (*important!*). If increase, this *might* increase accuracy but also training time by a lot.
